Library/Homebrew/cmd/update-report.rb

```diff
@@ -433,6 +445,19 @@ def dump
     dump_formula_report :DC, "Deleted Casks"
   end
 
+  def dump_without_updated_formulae
```

Comment:
Could this be a conditional argument on `dump(updated_formula_report: false)` instead? There's quite a bit of duplication here (and the `Key Legend` is missing).